### PR TITLE
Hello microverse project pull request

### PR DESCRIPTION
**Pull request by igna10697**

### CODE REVIEW

**Comment by igna10697** [comment]
This is a simple html and css project

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
+</html>
```
Comment (iambenkis) [inline index.html:13]: - [ ] Keep doing the great work :clap:

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: brown; 
+}
```
Comment (iambenkis) [inline style.css:3]: - [ ] Keep doing the great work :clap:

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @igna10697  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉 

- Gitflow is followed. :heavy_check_mark:
- Well implemented HTML and CSS. :heavy_check_mark: 
- Meaningful commit sentences :heavy_check_mark: 



 ## Required Changes ♻️

- [ ] Good job @igna10697   :+1:  setting everything up, but your PR summary should be more description as required . :100: , Kindly follow this [example](https://github.com/iambenkis/Space-X-/pull/32) to see what it looks like. And update yours.

Screenshot from 2023-01-11 10-16-19

- [ ] I acknowledge your effort setting up linters :pray: , unfortunately all the linters you set should have passed. Kindly make sure all your linters passed :100: 

I suggest you to run this command to solve them : 
```bash
npx stylelint css/styles.css --fix
```
For more information about style lint look [here](https://medium.com/@mrjohnkilonzi/how-to-css-lint-auto-fix-using-stickler-and-stylelint-37b52693d449)

- [ ] I noticed that your README file is not on the correct branch, and when you are going to merge your branches, the content that will be considered is the content of the README file on the `htmlcss_branch` branch. Kindly make sure you have a well documented README file on the feature branch. :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NduatiKagiri — COMMENTED** [review]
Hi @igna10697 , 

# INVALID REVIEW REQUEST ❌ ❌ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where some Linters checks are not passing, and you have not made changes to your README as requested by the previous reviewer.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**